### DISCUSSION TITLE
Desktop Shortcut

### DISCUSSION THREAD

**Discussion by akilou** [filips123/PWAsForFirefox#250] [Q&A] [answered]
Can I make a desktop shortcut for a specific site in Windows? I see that I can pin the icon to the task bar, but can I put a shortcut on my desktop?

**Comment by filips123** [comment, marked as answer]
Yes. Find the shortcut in your start menu, right click on it, select "More" and "Open file location". Then, in the explorer window that will open, simply copy shortcut for that web app to your desktop.
